pkg/deploy/role.go

```diff
@@ -23,13 +23,74 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+func SyncTLSRoleToCluster(deployContext *DeployContext) (*rbac.Role, error) {
+	tlsPolicyRule := []rbac.PolicyRule{
+		{
+			APIGroups: []string{
+				"",
+			},
+			Resources: []string{
+				"secrets",
+			},
+			Verbs:     []string{
+				"create",
+			},
+		},
+	}
+	return SyncRoleToCluster(deployContext, "che-tls-job-role", tlsPolicyRule)	
```

Comment:
use the const
https://github.com/svor/che-operator/blob/sv/metrics/pkg/deploy/tls.go#L39